Gray Code.cpp

```diff
@@ -0,0 +1,55 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+void Gray_Code(int n)
+{
+    if(n <= 0)
+    {
+        cout << "Invalid input length";
+        return;
+    }
+    vector<string> v;
+    v.push_back("0");
+    v.push_back("1");
+    for(int i = 2; i < (1 << n); i <<= 1)
+    {
+        for(int j = i-1; j >= 0; j--)
+	    v.push_back(v[j]);
+        for(int j = 0; j < i;    j++)
+	    v[j] = "0" + v[j];
```

Comment:
inconsistency in indentation

Reply:
Can you please open the code file in my branch.I have properly indented it but it's not showing in the files changed section.Kindly see the actual file and then review